main/SetupMenu.cpp

```diff
@@ -980,12 +980,44 @@ void SetupMenu::audio_menu_create_volume( MenuEntry *top ){
 	top->addEntry( amspvol );
 }
 
+void SetupMenu::audio_menu_create_mute( MenuEntry *top ){
+	SetupMenuSelect * asida = new SetupMenuSelect( PROGMEM"In Sink", RST_NONE, 0 , true, &audio_mute_sink );
+	asida->setHelp(PROGMEM"Select whether vario audio will be muted while in sink");
+	asida->addEntry( PROGMEM"Stay On");  // 0
+	asida->addEntry( PROGMEM"Mute");     // 1
+	top->addEntry( asida );
+
+	SetupMenuSelect * ameda = new SetupMenuSelect( PROGMEM"In Setup", RST_NONE, 0 , true, &audio_mute_menu );
```

Comment:
guess this is redundant to already defined "audio_disable" variable, why we need a second setup element here?
Note by renaming this, an in service upgrade will fail, pilots using already the above will fall back to the new default, even they had configured it.